draft/index.html

```diff
@@ -481,42 +481,44 @@ <h2>Reconciliation Queries</h2>
-        These are used to further refine the list of candidates by allowing clients to specify other attributes of entities, beyond their name in the <code>query</code> field.
-        A reconciliation service that supports properties SHOULD provide a <a>suggest service</a> for discovering these properties.</p>
-        <p>In addition to <code>pid</code> and <code>v</code>, services MAY support the following optional fields that allow clients to specify the effect of each property on the resulting list of candidates.
-          If these fields are omitted, the exact behavior ("must match all", "should match some", etc.) is up to the service.</p>
-        <dl>
-          <dt><code>required</code></dt>
-          <dd>A boolean indicating if a match for the property is required for an entity to enter the list of candidates (i.e. acting like a filter or a WHERE clause in SQL)
-            or optional (i.e. only effecting the entity's rank in the list of candidates);</dd>
-          <dt><code>match_quantifier</code></dt>
-          <dd>A string to indicate which of the values in <code>v</code> to match. MUST be "any" (equivalent to boolean OR), "all" (equivalent to boolean AND), or "none" (equivalent to boolean NOT);</dd>
-          <dt><code>match_qualifier</code></dt>
-          <dd>A string to indicate how to match the values in <code>v</code>.
-            This can be used for general matching relations like "skos:exactMatch", "skos:closeMatch", etc. or for specific features like spatial matching with geo data
-            (e.g. containment search with "schema:containsPlace" etc.) or custom matching on date fields (e.g. services supporting the [[EDTF]] specification could use "EDTF:Level-0" etc.
-            To allow discovery of supported qualifiers by clients, services that support <code>match_qualifier</code> MUST return the supported <code>match_qualifiers</code> for each property
-            in their property <a href='#suggest-responses'>suggest responses</a>.</p></dd>
-        </dl>
-        </dd>
-        </dl>
+            <dd>An optional limit on the number of candidates to return, which must be a positive integer;</dd>
+          </dl>
+        </p>
+        <p>
+          A <dfn>property assignment</dfn> specifies the expected value of a property on the entities to match.
+                These are used to filter the set of candidates (similar to a WHERE clause in SQL),
+		by allowing clients to specify an attribute of entities that should match. It consists of:
+         <dl>
+            <dt><code>pid</code></dt>
+            <dd>A <a href='#properties'>property</a> identifier. If this is not provided, then this signals that
+                the client intends to search for entities with similar names. The specifics of how this similarity
+                is defined are determined by the service.</dd> 
```

Comment:
Having a missing property signal a label/name search seems potentially error-prone.

Reply:
Why would it generate errors?  A default fallback strategy for "Things not Strings" would indeed be if Things (properties) are not provided then a String (name/label) is searched.  And so we accounted for that with query or properties must be supplied in the query syntax.  In the current draft:

> 4.1 Structure of a Reconciliation Query #structure-of-a-reconciliation-query
>A **reconciliation query** consists of the following fields. At least one of `query` or `properties` must be supplied, but all other fields are optional.
>
>**query**
>A query string, consisting of a non-empty string. By supplying such a string, a client intends to search for entities with similar names. The specifics of how this similarity is defined are determined by the service.

1. So maybe we just need to reword and cleanup 503-505 to align with what we already say at the top of the spec in `#structure-of-a-reconciliation-query`
2. capitalize MUST in the query structure wording - At least one of `query` or `properties` MUST be supplied, but all other fields are optional.

Reply:
In our monthly call we weighed the pros and cons of having a pre-defined id, mandated by the specs, versus other solutions and the group that was on the call converged towards the solution of not including a pid. The main reluctance I have towards using an id mandated by the specs is that it can conflict with the naming choices of other properties in the service. By using `rdfs:label` specifically I'd be a bit worried that we are sort of implying that this has something to do with RDF, which it does not (so far at least). To me it seems to be mostly a matter of taste, honestly. I am happy to adapt the PR to another consensus if there is one.

Reply:
@thadguidry 

>  But it was removed in this commit and now seems kinda important to state that somehow. Seems like we need to be explicit and mention if `query` is always required (MUST be supplied) inside properties. I don't see this clearly, and previously we had the anyOf.

I wonder if there's a misunderstanding here. This PR removes `query` as a special JSON keyword altogether. It just becomes another `v` entry under `properties`, with an empty `pid` to signal that it is expected to match names or label-like properties. On line 486, it is stated that `properties` is "An array of property assignments (at least one)." which is equivalent to the previous anyOf statement.

Reply:
@osma Misunderstanding indeed.  Sorry, and why I stopped commenting when I realized what was happening. :-)  I've made a note in our etherpad to maybe bring clarity further https://etherpad.lobid.org/p/reconciliation-january-2024#L32  I'll be on the meeting call later today.

Reply:
If that was the consensus of the expert group, I'm fine with it. The "error prone"ness that I was talking about was for developers who accidently forget to include an ID, not production scenarios, but they'll figure it out.